lib/utils/promote.ts

```diff
@@ -325,7 +325,7 @@ async function createApplication(
 ): Promise<BalenaSdk.Application> {
 	const form = await import('resin-cli-form');
 	const validation = await import('./validation');
-	const patterns = await import('./patterns');
+	const { printErrorMessage } = await import('../errors');
```

Comment:
The `errors` module is already imported at the top of the file, so `printErrorMessage` could be imported there too.

Reply:
Good catch.